free-programming-books-zh.md

```diff
@@ -466,27 +466,27 @@
 * [Nutz-book Nutz烹调向导](http://nutzbook.wendal.net)
 * [Nutz文档](https://nutzam.com/core/nutz_preface.html)
 * [REST 实战](https://github.com/waylau/rest-in-action)
-* [Spring Boot参考指南](https://github.com/qibaoguang/Spring-Boot-Reference-Guide) (翻译中)
+* [Spring Boot参考指南](https://github.com/qibaoguang/Spring-Boot-Reference-Guide) (:编著状态: *翻译中*)
 * [Spring Framework 4.x参考文档](https://github.com/waylau/spring-framework-4-reference)
 * [用jersey构建REST服务](https://github.com/waylau/RestDemo)
 
 
 ### Javascript
 
 * [Airbnb JavaScript 规范](https://github.com/adamlu/javascript-style-guide)
-* [ECMAScript 6 入门](http://es6.ruanyifeng.com) (作者：阮一峰)
+* [ECMAScript 6 入门](http://es6.ruanyifeng.com) - 阮一峰
 * [Google JavaScript 代码风格指南](http://bq69.com/blog/articles/script/868/google-javascript-style-guide.html)
 * [JavaScript Promise迷你书](http://liubin.github.io/promises-book/)
 * [Javascript 原理](https://web.archive.org/web/20170112164945/http://typeof.net/s/jsmech/)
 * [JavaScript 标准参考教程（alpha）](http://javascript.ruanyifeng.com)
-* [《JavaScript 模式》](https://github.com/jayli/javascript-patterns) “JavaScript patterns”中译本
+* [《JavaScript 模式》](https://github.com/jayli/javascript-patterns) (《JavaScript patterns》译本)
```

Comment:
Maybe we should discuss this, we could found original book name and writer in [github-repo](https://github.com/jayli/javascript-patterns), should we still keep in `-zh` file? because many of the `zh` books are translate, I think it not need to keep.

Reply:
Just personal suggestion:  for a Chinese book name I think we should use A CHINESE name, We can refer to the names of physical books. Another point of it, in the others version book list(日本語, or Spanish) , they are also use Local language book name, so I think we should use a local NAME, and leave a COMMENTS to mark origin name.

for leave a `zh` or  `-zh`  in a URL, it's unconspicuous and we can't see it in the preview markdown file, because the `zh` in URL.

Reply:
but many of resource in `free-programming-books-zh.md` are translate from English version. So I think we could remove origin name, or add origin name while other contributors add resource in `free-programming-books-zh.md`. what do you think?

Reply:
yes there are many translated books, for those books, I think it's better to add a origin author or a origin name.  In fact, there is no detailed description regarding **origin name** of a book in the CONTRIBUTING.md file. I think it's good to add it, and "Not add it" it's passable (*￣︶￣)

Reply:
I think so, but we should change both `CONTRIBUTING.md` and `CONTRIBUTING-zh.md`, I think  other language are in same situation.

Reply:
sure, I suggest we'd better to open an Issue to discuss it that is regarding how to describe origin name and author in  `CONTRIBUTING.md` and `CONTRIBUTING-zh.md`. 

For this PR , we can keep intact for the moment.

Reply:
agree with that